src/script/conversation/ConversationRepository.ts

```diff
@@ -2463,7 +2453,7 @@ export class ConversationRepository {
   /**
    * Clears conversation content from view and the database.
    *
-   * @param conversationEntity Conversation entity to delete
+   * @param conversationEntity Conversation entity to clear
    * @param timestamp Optional timestamps for which messages to remove
    */
   private async _clearConversation(conversationEntity: Conversation, timestamp?: number) {
```

Comment:
Shouldn't this one also be adjusted as well?

Reply:
Good catch, I've ended up removing this method and creating `onConversationContentCleared` that is fired when we receive member update event in self-conversation with information that conversation was cleared, so all other self clients also do the "clear". https://github.com/wireapp/wire-webapp/pull/16247/commits/16cee22b6e3014c11bd048e59313065e4902b291